app/controllers/bookings_controller.rb

```diff
@@ -1,8 +1,49 @@
 class BookingsController < ApplicationController
+  before_action :find_booking, only: [:show :edit, :update, :destroy]
+
+  def index
+    @bookings = Booking.all
+  end
 
   def new
+    @booking = Booking.new
   end
 
   def create
+    @booking = Booking.new(booking_params)
+    @booking.user = @user
```

Comment:
You need to find the user. According to the routes, the user should be in params[:user_id]